src/solver/optimisation/opt_construction_matrice_des_contraintes_cas_quadratique.cpp

```diff
@@ -32,6 +32,8 @@
 #include "../simulation/sim_extern_variables_globales.h"
 #include "opt_fonctions.h"
 
+using namespace Antares::Data;
```

Comment:
As the file barely changed, is `using namespace Antares::Data;` still required ?

Reply:
Indeed, removed it